mentor-mind-app/src/components/upload-document-popup/UploadDocumentPopup.tsx

```diff
@@ -0,0 +1,88 @@
+import {Box, Button, TextField, Typography, styled, FormControl, Select, MenuItem, Stack} from "@mui/material";
+import { useState } from "react";
+import PinnedContainer from "../containers/PinnedContainer";
+
+
+const CssTextField = styled(TextField)({
+    '& .MuiInputBase-root': {
+        backgroundColor: "var(--secondary-color)",
+        borderRadius: "10px"
+    }
+})
+const CssSelect = styled(FormControl)({
+    '& .MuiInputBase-root': {
+        backgroundColor: "var(--secondary-color)",
+        borderRadius: "10px"
+    }
+})
+
+
+const UploadDocumentPopup = () => {
+    const handleUploadDocument = () => {}
+    const [icon, setIcon] = useState("");
+
+    return (
+        <Stack direction={"row"} justifyContent={"space-evenly"} minHeight={"calc( 100% - 100px );"} height={"calc( 100% - 100px );"} alignItems={"center"} margin={"auto"}>
+            <PinnedContainer width="60%" height="85%">
+                <Box component="form" onSubmit={handleUploadDocument} noValidate
+                     sx={{
+                         flexDirection: 'column',
+                         display: 'flex',
+                         alignItems: 'center',
+                         height: 1,
+                         justifyContent: 'space-between'
+                     }}
+                >
+                    <Box sx={{
+                        margin: 5,
```

Comment:
![image](https://github.com/diss-mentor-mind/MentorMind/assets/72067145/8820ae9f-9aec-44af-baeb-8d3494724887)

For me it's overflowing a little, but you can reduce the bottom margin here (like `marginBottom: 2`) and that fixes it.

Reply:
Besides that, it looks good to me